dymos/transcriptions/transcription_base.py

```diff
@@ -864,7 +864,7 @@ def configure_objective(self, phase):
 
             from ..phase import Phase
             super(Phase, phase).add_objective(obj_path, ref=options['ref'], ref0=options['ref0'],
-                                              index=obj_index, adder=options['adder'],
+                                              index=obj_index, flat_indices=True, adder=options['adder'],
```

Comment:
Guess I missed this in the OpenMDAO PR, but I wonder if we should call it `flat_index` to go with `index`.  Then again, maybe it would be better to change `index` to `indices`, since some drivers do support multiple objectives (though I think we only support them as individual scalars.)  No action here, but I guess we should make a mental note.